Core/Steps/SubSteps/ReleaseVersionAndMoveIssuesSubStep.cs

```diff
@@ -66,99 +75,67 @@ public void Execute (SemanticVersion currentVersion, SemanticVersion nextVersion
       AddNewlyReleasedVersionToClosedIssuesOnlyAssociatedWithFullVersion(currentVersion);
   }
 
+  private bool ShouldMoveIssuesToNextVersion (string versionID, string nextVersionID, out IReadOnlyList<JiraToBeMovedIssue> issuesToMove)
+  {
+    if (versionID == nextVersionID)
+    {
+      issuesToMove = Array.Empty<JiraToBeMovedIssue>();
+      return false;
+    }
+
+    issuesToMove = JiraIssueService.FindAllNonClosedIssues(versionID);
+    if (issuesToMove.Count == 0)
+      return false;
+
+    Console.WriteLine("These are some of the issues that will be moved by releasing the version on jira:");
+    PrintIssueListing(issuesToMove);
+    Console.WriteLine("Do you want to move these issues to the new version and release the old one or just release the old version?");
```

Comment:
what does YES and No mean, here. please reword or rebuild to multiple choice. Don't know which is better in the workflow without demo